modules/understanding-upgrade-channels.adoc

```diff
@@ -134,10 +134,10 @@ ifndef::openshift-origin[]
 [discrete]
 == Switching between channels
 
-A channel can be switched from the web console or through the `adm upgrade channel` command:
+A channel can be switched from the web console or through the `patch` command:
 
 ----
-$ oc adm upgrade channel clusterversion version --type json -p '[{"op": "add", "path": "/spec/channel", "value": "<channel>”}]'
+$ oc patch clusterversion version --type json -p '[{"op": "add", "path": "/spec/channel", "value": "<channel>"}]'
```

Comment:
Prior to 4.9, we can use the `patch` command to change the channel. With 4.9 and later, we can use `oc adm upgrade channel <channel>` to change the channel.

We had a discussion in https://github.com/openshift/openshift-docs/pull/36015#discussion_r705954438.